kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/DisbursementVoucherInvoiceNumberEnteredValidation.java

```diff
@@ -17,7 +17,6 @@
 
     @Override
     public boolean validate(AttributedDocumentEvent event) {
-        boolean superValidate = super.validate(event);

```

Comment:
Are we sure we're not missing additional validations that might be present in super?